Add makefile, azure-pipelines.yml (#9)
bcgsc/ntEdit: README.md

*We recommend running ntEdit in Mode 1 (or 0)

### ntedit-make
------------

> You can run nthits-ntedit pipeline with a makefile.

Reviewer: ```suggestion
The ntEdit pipeline can be run with a Makefile (`ntedit-make`), which will run ntHits, then ntEdit for you.
```

> <pre>

Reviewer: ```suggestion
```

> 	* This make file REQUIRES draft genome, read files(#>=1) and k value. It will automatically run ntEdit after ntHits to output the final polished scaffold file. 

Reviewer: ```suggestion
	* Inputs: Read file(s), draft genome to be polished, k-mer size (`k`)
	* Outputs: Polished scaffolds in FASTA format
```

> 	* If you submit more than one read files, specify their prefix with "reads=prefix" and make sure that they are all in acceptable formats(fastq, fasta, gz, bz, zip).

Reviewer: ```suggestion
	* If you submit more than one read file, ensure each file has the same prefix (ex. myReads1.fq.gz, myReads2.fq.gz, ...) and specify the prefix with "reads=prefix". Ensure each read file is in an acceptable format (fastq, fasta, gz, bz, zip)
```

> 	* You are suggested to specify cutoff parameter yourself or define solid=true in your command to set it automatically.

Reviewer: ```suggestion
	* We suggest either specifying the cutoff parameter or defining solid=true in your command to set it automatically.
```

> Example:	Files in directory are myDraft.fa myReads1.fq myReads2.fq

Reviewer: ```suggestion
Example usage:
Draft genome to be polished: myDraft.fa
Input reads: myReads1.fq, myReads2.fq
```

> 		./ntedit-make ntedit draft=myDraft.fa reads=myReads k=64 cutoff=2 or

Reviewer: ```suggestion
<pre>
		./ntedit-make ntedit draft=myDraft.fa reads=myReads k=64 cutoff=2 or
```

		./ntedit-make ntedit draft=myDraft.fa reads=myReads k=64 solid=true

> 	Usage: ./ntedit-make ntedit [OPTION=VALUE]...

Reviewer: You don't have to copy and paste the entire help message. You can have the above examples, and either highlight the most important ones, or make a reference to how to see the `help` page from the makefile.
Also, please make sure your formatting has the example code the 'pre' boxes only, not the entire new message (I tried to move it, but check that I've done that correctly)
